api/main.py

```diff
@@ -616,3 +617,32 @@ def root() -> object:
     )
 
     return {"navigationmenudata": query_sidebar_menu.result}
+
+@APP.get("/search/{search_string}")
+async def get_search_results(search_string: str):
+    """
+    Returns search results for given search string.
+    :return: List of search results
+    """
+    database = get_db()
+    result = []
+    search_string_length = len(search_string)
+    if search_string_length > 6 and search_string_length < 150:
+        search_string_precise, search_string_fuzzy = search_utils.preprocess_search_string(search_string)
+
+
+        query_search = database.AQLQuery(
+            query=search_queries.QUERY_SEARCH, bindVars={"search_string": search_string_precise, "search_string_fuzzy": search_string_fuzzy},batchSize=300, rawResults=True       
+        )
+        query_result = query_search.result[0][:300]
+    result = search_utils.postprocess_results(search_string_precise,query_result)
+    return {"searchResults": result}
+
+@APP.get("/sanskrittagger/{sanskrit_string}")
```

Comment:
hmm, could you explain what does this function do? Could we possibly do it in the frontend?

minor: we should probably use a hyphen here: `sanskrit-tagger`.

Reply:
Which function do you mean? postprocess_results? 
I agree that renaming to sanskrit-tagger is a good idea, will do that.

Reply:
sorry, I meant `search_utils.tag_sanskrit`.

Reply:
which function you mean precisely?

Reply:
ah ok! tag_sanskrit calls the neural network from the package and splits sandhi and stems it so continuous strings like ātmadharmopacāro are turned into ātman dharma upacāra which are handy for indexing in the database.

Reply:
It can also do a bit more like add tags (gender, number, grammatical function etc). I created this endpoint on the backend because I have some plan for the future to connect this to a tool on the BN frontend. So people can use it as a standalone application.

Reply:
thanks for the explanation @sebastian-nehrdich , sounds great!